spec/graphql-crystal/language/generation_spec.cr

```diff
@@ -230,7 +230,7 @@ describe GraphQL::Language::Generation do
           expected = <<-schema
             type Foo {
               one(argument: String): Type
-              two(argument: Color = Red): Type
+              two(argument: Color = "Red"): Type
```

Comment:
has the graphql spec changed? This is an enum and should work without quotes right?

Reply:
You're right, it should be enum value. I'll see what i can do.

Reply:
fixed enum value parsing